src/uct/ib/rdmacm/rdmacm_iface.c

```diff
@@ -300,8 +299,8 @@ static UCS_CLASS_INIT_FUNC(uct_rdmacm_iface_t, uct_md_h md, uct_worker_h worker,
                            const uct_iface_config_t *tl_config)
 {
     uct_rdmacm_iface_config_t *config = ucs_derived_of(tl_config, uct_rdmacm_iface_config_t);
-    size_t ip_len = ucs_max(INET_ADDRSTRLEN, INET6_ADDRSTRLEN);
-    char *ip_str  = ucs_alloca(ip_len);
+    size_t ip_port_len = ucs_max(INET_ADDRSTRLEN, INET6_ADDRSTRLEN) + 1 + sizeof(in_port_t);
```

Comment:
sizeof(in_port_t) is not relevant .. probably need something like log10(SHORT_MAX).. i would just add a reasonably large constant in rdmacm and use it instead of duplicating this expression in all places which need to generate debug/error messages

Reply:
i will add the constant in ucs since i need to use it in ucp as well.